qa/vagrant/build.gradle

```diff
@@ -64,7 +64,8 @@ String upgradeFromVersion
 
 String maybeTestsSeed = System.getProperty("tests.seed", null);
 if (maybeTestsSeed != null) {
-  seed = new BigInteger(maybeTestsSeed, 16).longValue()
+  def (masterSeed, testSeed) = maybeTestsSeed.tokenize(':')
+  seed = new BigInteger(masterSeed, masterSeed.size()).longValue()
```

Comment:
I'm not sure, this will try to parse "abc:def" as a ternary value and that will fail. Also, shouldn't it be `masterSeed.length())`?